README.md

```diff
@@ -167,3 +167,16 @@ func (r *repo) Save(ctx context.Context, u *user) error {
 
 * To install all dependencies use `make go.mod.tidy` or `make go.mod.vendor`.
 * To run all tests use `make go.test` or `make go.test.with_real_db` for integration tests.
+
+### Requirements
+
+- [golangci-lint](https://golangci-lint.run/welcome/install/)
+
+### Local Running
+
+To run database by docker, there is [docker-compose.yaml](trm/drivers/test/docker-compose.yaml).
+```bash
+docker compose -f trm/drivers/test/docker-compose.yaml up
```

Comment:
Prove that old version vulnerabilities will not happen in user code.

Reply:
added here 

[trm/internal/prove_receiving_updates_without_vulnerabilities/README.md](https://github.com/avito-tech/go-transaction-manager/pull/107/files#diff-57dea7d98e4da9e094f098dff3a4bbef23ab16550862c26b9c7e245fa914ad1a)